Shared/Features/Freaks/ViewModel/FreaksViewModel.swift

```diff
@@ -6,6 +6,20 @@ final class FreaksViewModel: ObservableObject {
 
     @Published var skillSelections: [Skill] = []
     @Published var shouldFilterFreaks: Bool = false
+    
+    var skillsButton = String()
```

Comment:
It’s the text for the button. I named it skillsButton because I thought of using it for skills button. I initiate it with String because i am adding text in this variable